src/template.js

```diff
@@ -0,0 +1,64 @@
+var globby = require('globby');
+var path  = require('path');
+var utils = require('./utils');
+
+/**
+ * Register helpers on the passed Handlebars instance.
+ * Accept an object with helperKey => helperFunctions,
+ * or a glob (as Array or string) of files (modules) to
+ * register. In the latter case, the filename w/o extension
+ * will be used as the helper key.
+ *
+ * @param {Object} Handlebars handlebars instance
+ * @param {Object} || [{Array} || {String} glob]
+ * @param return {Promise}, resolving to
+ *           {Object} of all helpers registered on Handlebars
+ */
+function prepareHelpers (Handlebars, helpers = {}) {
+  if (typeof helpers === 'string' || Array.isArray(helpers)) {
+    return globby(helpers).then(helperPaths => {
```

Comment:
I see that `preparePartials ` uses `utils.readFiles` for something similar to this. Can this be abstracted into a utility function as well?